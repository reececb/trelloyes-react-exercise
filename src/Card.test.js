import React from 'react';
import renderer from 'react-test-renderer';
import Card from "./Card";
import ReactDOM from 'react-dom';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Card />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title="Card" content="Data"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });