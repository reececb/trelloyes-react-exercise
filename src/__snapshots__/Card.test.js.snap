// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  classname="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Card
  </h3>
  <p>
    Data
  </p>
</div>
`;
